doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+==================================
+Gas-Solid Spouted Cylinder Bed
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for a detailed description on the concepts and physical meanings of the DEM and CFD-DEM parameters.
+This example is an extension of the `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ for a cylindrical geometry. The boundary condition and some parameters have been changed accordingly.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a solid-gas cylinder-shaped spouted bed
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed``).
+
+- Parameter file for CFD-DEM simulation of the spouted bed: ``gas-solid-spouted-bed.prm``
+- Parameter file for particle generation and packing: ``dem-packing-in-spouted-bed.prm``
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the spouting of spherical particles in air in a cylindrical geometry. As noted in the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-beml>`_, we use ``lethe-particles`` to fill the bed with particles, and ``lethe-fluid-particles`` as a solver of CFD-DEM.
+
+-------------------
+DEM Parameter File
+-------------------
+
+Here, we will focus only on the parts that have been modified.
```

Comment:
That have been modified compared to the pseudo-2D spouted bed? If that is the case, please be more explicit

Reply:
Yes, that's what I meant. I mentioned this in line 6, the beginning of this document. 
would you say I should mention it again here?